lib/collections/discussions/tags/schema.js

```diff
@@ -0,0 +1,16 @@
+var Schemas = {};
+
+Schemas.Tag = new SimpleSchema({
+    name: {
+        type: String,
+        label: 'name',
+    },
+    type: {
+        type: String,
+        label: 'Type',
+        allowedValues: ['Lectures', 'Labs', 'Project', 'Topic'],
```

Comment:
- the conventions are a little off here .. some words are plural and some are singular. (I understand the reasoning behind it though) but still I think consistency is better .
- also Capitalized words are not the convention for constants .. it's either lower-case or UPPERCASE. we were using lower case so I suggest this could be changed  to 
```js
 allowedValues: ['lecture', 'lab', 'project', 'topic']
```